src/Pages/Applications/DevtronApps/Details/AppConfigurations/AppComposeRouter.tsx

```diff
@@ -123,17 +125,17 @@ export default function AppComposeRouter({
                     <Route
                         key={`${path}/${URLS.APP_CM_CONFIG}`}
                         path={`${path}/${URLS.APP_CM_CONFIG}/:name?`}
-                        render={(props) => <ConfigMapList isJobView={isJobView} isProtected={false} />}
+                        render={() => <ConfigMapList isJobView={isJobView} isProtected={false} />}
```

Comment:
We should change it to 
`<Route
                        key={`${path}/${URLS.APP_CM_CONFIG}`}
                        path={`${path}/${URLS.APP_CM_CONFIG}/:name?`} >
                        <ConfigMapList isJobView={isJobView} isProtected={false} />
                        </Route>`